spec/player_spec.rb

```diff
@@ -0,0 +1,11 @@
+require_relative '../lib/board'
+require_relative '../lib/player'
+
+RSpec.describe Player do
+  let(:board) { Board.new(3) }
```

Comment:
Looks like this board is not necessary for this test anymore?

Reply:
Resolved at 2b50818af7b22a53048756aedeb2e0a3a83b1838